libfwupdplugin/fu-archive.c

```diff
@@ -70,23 +72,23 @@ fu_archive_init(FuArchive *self)
 GBytes *
 fu_archive_lookup_by_fn(FuArchive *self, const gchar *fn, GError **error)
 {
-	GBytes *fw;
+	GBytes *bytes;
```

Comment:
I will revert that change as well, and/or propose in another commit.

`fw` does not make sense in the `FuArchive`.

Reply:
agree. `blob` is another name for a GBytes we use.